cli/src/main/java/com/box/l10n/mojito/cli/command/CommitCreateCommand.java

```diff
@@ -0,0 +1,90 @@
+package com.box.l10n.mojito.cli.command;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.command.param.Param;
+import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.rest.client.CommitClient;
+import com.box.l10n.mojito.rest.client.RepositoryClient;
+import com.box.l10n.mojito.rest.entity.Commit;
+import com.box.l10n.mojito.rest.entity.Repository;
+import org.fusesource.jansi.Ansi;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author garion
+ */
+@Component
+@Scope("prototype")
+@Parameters(commandNames = {"commit-create", "cc"}, commandDescription = "Create commit information in Mojito.")
```

Comment:
haha, not sure about the short name in that case. I've just drop it for some commands